<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="2" column="1">
     <widget class="QLabel" name="denemeLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="denemeEkle">
      <property name="text">
       <string>Ekle</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Kesin Olanlar</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Ders Adı:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QLineEdit" name="kesinDersAd"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Ortalama:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Kredi:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QDoubleSpinBox" name="kesinKredi"/>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="QComboBox" name="kesinOrtalama">
           <item>
            <property name="text">
             <string>AA</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>BA</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>BB</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>CB</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>CC</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>DC</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>DD</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FD</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FF</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Deneme</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Ders Adı:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QLineEdit" name="lineEdit_2"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Ortalama:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Kredi:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_2"/>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="QComboBox" name="comboBox_2">
           <item>
            <property name="text">
             <string>AA</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>BA</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>BB</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>CB</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>CC</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>DC</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>DD</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FD</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FF</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="kesinLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="kesinEkle">
      <property name="text">
       <string>Ekle</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
